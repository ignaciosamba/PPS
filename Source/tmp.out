
C51 COMPILER V9.53.0.0 - SN: Eval Version
COPYRIGHT Copyright (C) 2012 - 2014 ARM Ltd and ARM Germany GmbH. All rights reserved.
*** WARNING C276 IN LINE 288 OF adc.c: constant in condition expression
*** WARNING C294 IN LINE 179 OF adc.c: unreachable code

C51 COMPILATION COMPLETE.  2 WARNING(S),  0 ERROR(S)
