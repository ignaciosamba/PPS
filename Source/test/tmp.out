
C51 COMPILER V9.53.0.0 - SN: Eval Version
COPYRIGHT Copyright (C) 2012 - 2014 ARM Ltd and ARM Germany GmbH. All rights reserved.
*** WARNING C318 IN LINE 3 OF test_global.c: can't open file '../fuentes/configuracion.h'
*** WARNING C206 IN LINE 17 OF test_global.c: 'iniciar_puertos': missing function-prototype
*** WARNING C206 IN LINE 18 OF test_global.c: 'iniciar_sysclock': missing function-prototype
*** WARNING C206 IN LINE 19 OF test_global.c: 'iniciar_UART': missing function-prototype

C51 COMPILATION COMPLETE.  4 WARNING(S),  0 ERROR(S)
